tests/test_extract_tts_spectrograms.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
I guess I couldn't explain well enough. I was suggesting to test ` TTS/bin/extract_tts_spectrograms.py` directly not the model. So if something fails in ` TTS/bin/extract_tts_spectrograms.py` we should know about it. In this case, it is not possible. 

You have two options. 

One is to create a shell script like the those under  "tests/bash_tests/*"

Or something like `test_synthesize.py` to call the script in python using `run_cli`. 

Makes sense ?

Reply:
So I have to train the models and use the checkpoint to extract the spectrograms. Should I add this test within test_glow-tts_train.sh and test_tacotron_train.sh to avoid training the model more than once during the test? Or is it better to create a separate script and train the TTS models and after extract the spectrograms?

Also, should I remove this test or is it better to remove it?

Reply:
You can just create dummy models without training and save them. Then load them and use with `extract_tts_spectrograms.py`.  Something like `TTS/tests/test_synthesizer.py`. You don't need to train them. 

Or you can use one of the release models to test the script. Either way is fine. 

In this script currently, you are basically testing glow TTS and the Tacotron model. So I think you can remove this.

Reply:
Thanks for the explanation, I was confused as my fork did not have `run_cli` and `TTS/tests/test_synthesize.py `so I was looking at `TTS/tests/test_synthesizer.py` and was not understanding exactly what you wanted. Verify Now :).